spinalcordtoolbox/scripts/sct_qc.py

```diff
@@ -47,6 +47,11 @@ def get_parser():
                         help='Plane of the output QC. Only relevant for -p sct_deepseg_lesion.',
                         choices=('axial', 'sagittal'),
                         required=False)
+    parser.add_argument('-exclude-label-text',
```

Comment:
I like to avoid negative options, because they tend to cause double negatives down the line, which are confusing (as you noted in your commit description). Could I suggest `-text-labels 0/1` instead, with a default of `1`? This would also be consistent with the rest of SCT, for example `sct_convert -squeeze` and `sct_compute_compression -normalize-hc` and the ubiquitous `sct_* -r` for removing temporary files.

Reply:
Ahhh, good thinking! That sounds good. :)